ui/src/components/common/FeedbackLink.tsx

```diff
@@ -0,0 +1,27 @@
+import { createFeedbackLink, openFeedbackWindow } from "../../utils/feedback";
+import { PaperId } from "../../state";
+
+import React from "react";
+
+interface Props {
+  text: string;
+  paperId?: PaperId;
+  extraContext?: object;
+}
+
+/**
+ * Controls the visual appearance of the button. The toolbar variant mimics
+ * the appearance of buttons rendered by the default PDF viewing experience.
+ */
+
+const FeedbackLink = ({ text, extraContext, paperId }: Props) => {
+  return (
+    <a href="#" onClick={() =>
+      openFeedbackWindow(createFeedbackLink(paperId, extraContext))
+    }>
```

Comment:
![tenor-6168103](https://user-images.githubusercontent.com/39743357/119706306-aa3ee400-be0e-11eb-8801-f78d61ea370a.gif)

Is there no link for this...link?